.github/workflows/wheel_macOS.yml

```
@@ -0,0 +1,66 @@
+name: wheel - macOS
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  wheel_macos:
+    name: macOS wheel
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+        python-version: [3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: check C++ compiler and Python
+        run: |
+          clang --version
+          python --version
+      - name: Install dependencies
+        run: |
+          python -m pip install --no-cache-dir --upgrade pip
+          python -m pip install --no-cache-dir "poetry<2,>=1.0"
+          python -m pip install --no-cache-dir "mypy-protobuf~=1.0"
+      - name: Build libraries
+        run: |
+          make
+      - name: Build wheel
+        run: |
+          python -m poetry build -v
+      - name: Fix wheel
+        run: |
+          python -m pip install wheel
+          python -m pip install delocate
+          delocate-wheel -v dist/*.whl -w ./wheelhouse
+          delocate-addplat -x 10_9_intel -x 10_10_intel .wheelhouse/*.whl
```

```suggestion
          delocate-addplat -x 10_9_intel -x 10_10_intel wheelhouse/*.whl
```
looks like there was an extra period